docs/contribute/modules/index.md

```diff
@@ -27,9 +27,9 @@ After setting up your repository and linking it to your GitHub account, you can
 You may implement your code in the ``init.py`` located in the respective feature folder in ``src/autora``.
 
 Please refer to the following guides on implementing
-- [theorists](theorist.md)
-- [experimentalists](experimentalist.md)
-- [experiment runners](experiment-runner.md)
+* [theorists](theorist.md)
```

Comment:
I think you need a new line here to separate the bullet point block from the paragraph above - otherwise it won't be rendered as a bullet. It's the same problem as we had with the admonitions.

```suggestion

* [theorists](theorist.md)
```